sage_modabvar/torsion_subgroup.py

```diff
@@ -203,13 +216,18 @@ def order(self):
             sage: J = J1(13)
             sage: J.rational_torsion_subgroup().order()
             19
-        """
-        try:
-            return self._order
-        except AttributeError:
-            pass
 
-        O = self.possible_orders()
+        Sometimes the order can only be computed with proof=False. ::
+
+            sage: J = J1(23)
+            sage: J.rational_torsion_subgroup().order()
+            Traceback (most recent call last):
+            ...
+            RuntimeError: Unable to compute order of torsion subgroup (it is in [408991, 9406793])
+
+
```

Comment:
Right here there should be the same example with proof=False, right?   `J.rational_torsion_subgroup().order(proof=False)`